utils/miscellaneous.py

```diff
@@ -19,8 +19,17 @@ def build_image_root_by_id(session_id):
 
 
 def validate_request_session(request):
+    # TODO Keep track of used session ids and check if current session id is valid
```

Comment:
I think we have two possibilities to overcome this problem:

1.  First we use the `check_image_exists` functions but then images will not be deleted as we can not determine wheter they are currently in use or just old pictures. We could check the modification date of each session present in the and then delete the older ones.
2. Second we implement a sort of middleware or a global state, which takes care of the cleanup.